### DISCUSSION TITLE
Campo contentUrl ausente en data de PBMessage

### DISCUSSION THREAD

**Discussion by Yiakman** [Cotalker/documentation#187] [2. Questions and Answers] [answered]
Necesito mandar un message con text/html y lo ideal es hacerlo mediante el campo contentUrl. Mi problema radica en que PBMessage no posee ese campo como data input. Hay razones a nivel de producto para omitirlo? De ser así, podemos dejarlo oculto y sin PBDoc o bien detrás de algún permiso especial?

**Comment by acedward** [comment, marked as answer]
1. Esta semana pasamos a productivo : https://doc.cotalker.com/blog/2021/04/28/bot-versions/
2. Esto nos permite, finalmente mejorar los bots - y agregar parámetros faltantes.
3. Podríamos agergar esta llave opcional en  https://github.com/Cotalker/documentation/blob/26e1afb50c73875444a5c776bb7e55ba2ec3316b/docs/documentation/automation/bots/pbmessage.md ó crear un fork del bot porque debería pedir menos campos

**Reply by Yiakman** [reply]
Único detalle es que lo necesito usar para [REDACTED], pero creo que tengo un hack

**Reply by acedward** [reply]
¿Cual es el problema?
